specification/metrics/effectivecost.md

```diff
@@ -1,13 +1,13 @@
 # Effective Cost
 
-Effective cost represents the sum of the amortized upfront fees, amortized recurring fees, and the usage cost, including discounts,  of the line item. This column is often used to show spending trends that include on demand rates, effective commitment-based discount rates, recurring commitments, as well as amortization of upfront fees. The fee and upfront amortization portion of this column's value should be proportional to the number of units for the line item and the time granularity of the data. The currency for the value specified in Effective Cost can be found in the [Billing Currency](#billingcurrency) column. The aggregated Effective Cost for a billing period may not match the charge received on the invoice for the same billing period.
+The Effective Cost represents a charge inclusive of the impacts of all reduced rates and discounts, augmented with the amortization of upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends. The amortization of upfront charges portion of this metric's value should be proportional to the [Quantity In Pricing Unit](#quantityinpricingunit) and the time granularity of the data.
 
 Practitioners are faced with two main challenges with respect to this metric:
```

Comment:
I understand the challenges are actually resolved thanks to the Effective cost.

Maybe:

< `Practitioners are faced with two main challenges with respect to this metric:`
> `This metrics resolves two challenges that are faced by practionners:`